lib/matplotlib/tests/test_image.py

```diff
@@ -713,7 +713,7 @@ def test_minimized_rasterized():
 
 
 def test_load_from_url():
-    path = Path(__file__).parent / "baseline_images/test_image/imshow.png"
+    path = Path(__file__).parent / "baseline_images/pngsuite/basn3p04.png"
```

Comment:
I don't understand why we think this image will be permanent either, and it seems a very obscure image to choose.  Isn't there something in mpl_data/images that would be more appropriate?

Reply:
This image will remain as it is also used in https://github.com/matplotlib/matplotlib/blob/50a535c2453f438a5bfbfc3bc7275c8dafc2f422/lib/matplotlib/tests/test_image.py#L151. 
Secondly, the `basn3p04.png` image is not used in `image_comparison` tests. So, we don't need to shift it into the `mpl_baseline_images` package in #17557

Reply:
But shouldn’t that be moved as well?  It’s going to be pretty confusing to have baseline_images and another package of baseline_images.

Reply:
The primary reason the `basn*.png` files exist is as input for the test in `test_png` so is not in the category of images that will be removed by the GSOC code landing.  On the other hand, the file that this used to point is a baseline image that will be removed.